docs/Constraints.md

```diff
@@ -121,4 +137,66 @@ Supported in .NET SDK and Visual Studio
            ]
        },
    }
+```
+
+## Installed Workloads
+Defines applicability of a template in the dotnet runtime with specific [workloads](https://github.com/dotnet/designs/blob/main/accepted/2020/workloads/workloads.md) installed.
+All the installed (queryable via `dotnet worload list`) as well as [extended](https://github.com/dotnet/designs/blob/main/accepted/2020/workloads/workload-manifest.md#workload-composition) workloads are inspected during evaluating of this constraint.
+
+**Configuration:**
+
+- `type`: `workload`
+- `args` (string, array). Mandatory. List of names of supported workloads (running host need to have at least one of the requested workloads installed).
```

Comment:
nit: if there is a list or command that shows list of available workloads, it would be good to include it here.
E.g. `To see the list of available workloads and their names, see <link>.`